### PR TITLE
Leaderboard hit the API

### PR DESCRIPTION
**Pull request by jheart-vic**
-  I created a new game with the name of your choice by using the API.
- I Implemented the "Refresh" button (receiving data from the API and parsing the JSON).
- I Implemented the form "Submit" button (sending data to the API).
- I Use `async` and `await` JavaScript features to consume the API.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    message.style.color = 'red';
+    setTimeout(() => {
+      message.style.display = 'none';
+    }, 2000);
+  }
+  await setScores(playerName.value, playerScore.value);
+  showAllScores();
+  playerName.value = '';
+  playerScore.value = '';
+});
```
Comment (vechicin) [inline src/index.js]: While the overall features are implemented correctly, they are not properly distributed in the UI. According to the project requirements, the `Submit` button should only `POST` to the API. When the user clicks the `Refresh` button is when the scores should be updated in the UI. Please keep your `showAllScores()` function strictly for the `Refresh` button.

Screen Shot 2022-06-15 at 16 15 40

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @jheart-vic ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights ⚡ 

- [x] Good job communicating with the API
- [x] Excellent job implementing custom validation!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by vechicin** [comment]
Hi, @jheart-vic I just noticed you introduced some changes to your review and now your linters are failing. Please make sure you don't submit your project for review unless you're 100% sure you don't want to introduce more changes because it complicates the reviewing process. Fix the linters as well, please.

File: modules/apiFlie.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+//   })
+// })
+// .then((data) => {
+//   data.json().then((val) => console.log(val))
+// })
+// getting scores
+// fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/EoTxCV02Ftb3pgz9Cebz/scores')
+// .then((data) => {
+//   data.json().then((val) => console.log(val))
+// })
```
Comment (MahmoudBakr23) [inline modules/apiFlie.js:59]: ### Optional
- Please delete the unnecessary code for a cleaner codebase.

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @jheart-vic 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

**Highlights**
- Well done making the required changes from the previous reviewer

**Optional**
_Please see the comment under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**